src/NzbDrone.Core.Test/HealthCheck/Checks/DotnetVersionCheckFixture.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using NUnit.Framework;
+using NzbDrone.Common.EnvironmentInfo;
+using NzbDrone.Core.HealthCheck.Checks;
+using NzbDrone.Core.Test.Framework;
+
+namespace NzbDrone.Core.Test.HealthCheck.Checks
+{
+    [TestFixture]
+    public class DotnetVersionCheckFixture : CoreTest<DotnetVersionCheck>
+    {
+        private void GivenOutput(string version)
+        {
+            WindowsOnly();
+
+            Mocker.GetMock<IPlatformInfo>()
+                  .SetupGet(s => s.Version)
+                  .Returns(new Version(version));
+        }
+
+        [TestCase("4.7.2")]
+        [TestCase("4.8")]
+        public void should_return_ok(string version)
+        {
+            GivenOutput(version);
+
+            Subject.Check().ShouldBeOk();
+        }
+
+        [TestCase("4.6.2")]
+        [TestCase("4.7")]
+        [TestCase("4.7.1")]
+        public void should_return_notice(string version)
+        {
+            GivenOutput(version);
+
+            Subject.Check().ShouldBeNotice();
+        }
+
+        public void should_return_warning(string version)
```

Comment:
Only minor but this test has no cases. I guess you're keeping the warning option around in case you want to use it later? Not sure it's used at the moment?

Reply:
yup, at first I produced warnings for it, but the notice shows as blue in the UI so draws enough attention. I kept the function around for later use. It's mostly that I want to ramp up the pressure to upgrade, notice now, warning later.